plugins/kafka-backend/config.d.ts

```diff
@@ -15,8 +15,21 @@
  */
 export interface Config {
   kafka?: {
+    /**
+     * Client ID used to Backstage uses to identify when connecting to the Kafka cluster.
+     */
     clientId: string;
+    /**
+     * List of brokers in the Kafka cluster to connect to.
+     */
     brokers: string[];
+
+    /**
+     * Optional SSL connection parameters to connect to the cluster. Passed directly to Node tls.connect.
+     * See https://nodejs.org/dist/latest-v8.x/docs/api/tls.html#tls_tls_createsecurecontext_options
+     *
+     * @visibility secret
```

Comment:
Visibility only applies to the value it's applied to, and none of the chilldren, so it needs to be set specifically for the `key`, or each of them if you want them all to be secret. There's really no need to make them all secret though

Reply:
Got it. Didn't know that, thanks!